e2e-tests/IIoTPlatform-E2E-Tests/RegistryHelper.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests {
+
+    using IIoTPlatform_E2E_Tests.Config;
+    using Microsoft.Azure.Devices;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Helper for managing IoT Hub device registry.
+    /// </summary>
+    class RegistryHelper {
+
+        public RegistryHelper(IIoTHubConfig ioTHubConfig) {
+            _ioTHubConfig = ioTHubConfig ?? throw new ArgumentNullException(nameof(ioTHubConfig));
+
+            _registryManager = RegistryManager.CreateFromConnectionString(_ioTHubConfig.IoTHubConnectionString);
+        }
+
+        /// <summary>
+        /// Wait until IIoT modules are in Connected state.
+        /// </summary>
+        /// <param name="deviceId"></param>
+        /// <param name="deviceId"> IoT Edge device id </param>
+        /// <param name="moduleNames"> List of modules to wait for, defaults to ModuleNamesDefault if not specified </param>
+        /// <param name="ct"> Cancellation token </param>
+        /// <returns></returns>
+        public async Task WaitForIIoTModulesConnectedAsync(
+            string deviceId,
+            IEnumerable<string> moduleNames = null,
+            CancellationToken ct = default
```

Comment:
>CancellationToken ct = default [](start = 12, length = 30)

CancellationToken should be mandatory, we don't want to wait forever

Reply:
Made cancellation token mandatory for `WaitForIIoTModulesConnectedAsync()` method.